pubspec.yaml

```diff
@@ -14,3 +14,6 @@ dev_dependencies:
   test: ^1.6.4
   matcher: ^0.12.5
   pedantic: 1.8.0
+  build_runner: ^1.0.0
```

Comment:
Are all these necessary?

I can see you use `build_runner` in the travis file, but if hat depends on the other two, shouldn't they just get included automatically?

The fewer extra dependencies you add, the less we risk someone getting 1.9.0 instead of 1.9.1 because of an incompatible constraint.

Reply:
These are all dev dependencies only - and all are required to run web tests through the common infrastructure.

- build_runner is the core build system
- build_web_compilers adds the DDC/Dart2Js actions
- build_test adds builders that bootstrap your test files (this is a requirement of the `test` package).